app/controllers/api/businesses_controller.rb

```diff
@@ -19,8 +19,20 @@ def show
       @owner = @business.owner
     end
 
+    DistanceInKm = 10.0 # could be an api parameter
+    KMperDegree = 111.045
+    DistanceLatInDegrees = DistanceInKm / KMperDegree
```

Comment:
That's better. But in idiomatic Ruby, constants go to the top of a class and are written in uppercase letters only. I.e. `DISTANCE_IN_KM`

Reply:
please improve implementation as required when merging